### PR TITLE
Add French translations for 23xxx codes

### PR DESCRIPTION
**Pull request by markbrough**
Adds missing French translations for 23xxx codes (energy sector).

Source document from OECD:
https://www.oecd.org/fr/cad/stats/documentupload/DCD-DAC(2013)15-ADD1-FINAL-FRE.pdf

These codes were removed here (it appears with effect for reporting on 2014 flows, so I guess this would mean `2013-12-31`):
https://one.oecd.org/document/DCD/DAC/STAT(2013)3/en/pdf

There is also a helpful table at the back of that document that maps to new codes.

It's a bit annoying that the generation messes around with the ordering of attributes, so the diff is longer than it should be...

### CODE REVIEW

File: xml/Sector.xml [hunk 1]
```diff
@@ -1,4 +1,5 @@
-<codelist name="Sector" xml:lang="en" category-codelist="SectorCategory" complete="1" embedded="0" xmlns:budget-alignment="http://iatistandard.org/activity-standard/overview/country-budget-alignment/">
+<?xml version="1.0" ?>
```
Comment (andylolz) [inline xml/Sector.xml]: Seems reasonable to add an XML declaration, though perhaps we should separately update all codelists?

**Comment by markbrough** [comment]
Sounds good, and I agree. I'm gonna try fixing https://github.com/codeforIATI/translate-codelists/issues/1 by trying another approach to indentation, which will hopefully make the diff a bit cleaner.

**Comment by markbrough** [comment]
OK, I think this is looking much nicer now! When you generate the file using `translate.py`, the first line is still slightly differently ordered (the `xmlns:budget-alignment` attribute is at the start, otherwise everything is the same). Otherwise, there is no change.

**Comment by andylolz** [comment]
> (the xmlns:budget-alignment attribute is at the start, otherwise everything is the same)

^^ Oh, but it’s totally fine to move this, since it only exists in our forked version anyway.

**Pull request merged**